index.html

```diff
@@ -184,6 +184,23 @@ <h2 class="logo-scratchx">ScratchX</h2>
                         </section>
                     </li>
                     <li>
+                        <a href="?url=https://Robotec-CoderZ.github.io/he.js" data-action="load-url">
+                            <header>
+                                <h2>EV3 LEGO Education</h2>
```

Comment:
This sounds good to me. I'll send a courtesy email to the developers to confirm the name changes.

Thanks!